packages/inventory-compliance/src/ComplianceEmptyState.test.js

```diff
@@ -2,9 +2,37 @@ import React from 'react';
 import toJson from 'enzyme-to-json';
 import { shallow } from 'enzyme';
 import ComplianceEmptyState from './ComplianceEmptyState';
+import { useQuery } from '@apollo/react-hooks';
+jest.mock('@apollo/react-hooks');
+jest.mock('apollo-boost');
 
 describe('ComplianceEmptyState', () => {
-    it('expect to render without error', () => {
+    it('expect to render without error if no policies exist', () => {
+        useQuery.mockImplementation(() => ({
+            data: { profiles: { totalCount: 0 } }, error: false, loading: false
```

Comment:
To stay in line with the what the `useQuery` API returns, I would suggets to use `undefined` instead of `false`.